backend/app/core/validators.py

```
@@ -0,0 +1,58 @@
+"""
+SQL検証関数
+"""
+import re
+from typing import Tuple, Optional
+from .error_codes import (
+    VALIDATION_EMPTY_SQL, 
+    VALIDATION_SQL_TOO_LONG, 
+    VALIDATION_INVALID_SQL
+)
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)